ateam_joystick_control/src/joystick_control_node.cpp

```diff
@@ -88,19 +91,35 @@ class JoystickControlNode : public rclcpp::Node
   rclcpp::Subscription<sensor_msgs::msg::Joy>::SharedPtr joy_subscription_;
   rclcpp::node_interfaces::OnSetParametersCallbackHandle::SharedPtr parameter_callback_handle_;
 
+  void SendStop()
+  {
+    if (control_publisher_) {
+      ateam_msgs::msg::RobotMotionCommand command_message;
+      control_publisher_->publish(command_message);
+    }
+  }
+
   void CreatePublisher(const int robot_id)
   {
-    std::string topic = command_topic_template_;
-    const auto placeholder_pos = topic.find("{}");
-    if (placeholder_pos != std::string::npos) {
-      topic.replace(placeholder_pos, 2, std::to_string(robot_id));
+    if (robot_id == -1) {
+      control_publisher_.reset();
+    } else {
+      std::string topic = command_topic_template_;
+      const auto placeholder_pos = topic.find("{}");
+      if (placeholder_pos != std::string::npos) {
+        topic.replace(placeholder_pos, 2, std::to_string(robot_id));
```

Comment:
If there's a placeholder, it's just at the beginning of the string, I assume from the ending string position (2)? Why not use the variable we assign on line 108 + 2 instead of an absolute/magic number?